dvc/system.py

```diff
@@ -79,13 +44,11 @@ def _reflink_darwin(src, dst):
         LIBC_FALLBACK = "/usr/lib/libSystem.dylib"
         try:
             clib = ctypes.CDLL(LIBC)
-        except OSError as exc:
+        except (FileNotFoundError, PermissionError) as exc:
```

Comment:
I added `Permission Error` because of the `DEBUG` message: `"unable to access '{}'`.

Reply:
But the debug message is useful even for an unidentified error. I probably would rather not touch this one.